app/views/shared/_nav.html.haml

```diff
@@ -8,7 +8,8 @@
         %li= link_to t('application.my_dashboard'), root_path, class: class_for_path(request, root_path)
       - if current_user && current_user.admin?
         %li= link_to t('application.recent_activity'), recent_activity_path, class: class_for_path(request, recent_activity_path)
-      %li= link_to t('application.training'), training_path, class: class_for_path(request, training_path)
+      - if !ENV['disable_training']
```

Comment:
Because of the way ENV gets populated from application.yml, the ENV variables are all strings. For other such feature switches, the strategy we use for checking them to check for the exact string that signifies the relevant state. So in this case:
```ruby
unless ENV['disable_training'] == 'true'
```
would be more consistent with the rest of the codebase.

In this case, the setting specified in application.example.yml will mean that `ENV['disable_training']` is the string `'false'` rather than the boolean, which means it won't behave as intended.